mergify_engine/redis_utils.py

```diff
@@ -93,9 +94,13 @@ async def load_scripts(redis: typing.Union[RedisCache, RedisStream]) -> None:
     scripts = list(SCRIPTS.items())  # order matter for zip bellow
     shas = [sha for _, (sha, _) in scripts]
     ids = [_id for _id, _ in scripts]
-    exists = await redis.script_exists(*shas)  # type: ignore[no-untyped-call]
+    await redis.on_connect()
+    await redis.send_command("SCRIPT EXISTS", *shas)
```

Comment:
what's the rational for not using `script_exists` anymore?

Reply:
same as the upper explanation